### PR TITLE
Move to dotnet6

### PR DESCRIPTION
**Pull request by JustRoxy**

### CODE REVIEW

File: backend/CircleForms/CircleForms.csproj [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
     <PropertyGroup>
-        <TargetFramework>net5.0</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
         <UserSecretsId>dceba1f4-5b57-42a6-8781-163388fd36a7</UserSecretsId>
         <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
         <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
+        <LangVersion>10</LangVersion>
```
Comment (stanriders) [inline backend/CircleForms/CircleForms.csproj:8]: `latest`

File: backend/CircleForms/Models/MongoDbRange.cs [hunk 2]
```diff
@@ -1,14 +1,13 @@
-﻿namespace CircleForms.Models
+﻿namespace CircleForms.Models;
+
+public class MongoDbRange
```
Comment (stanriders) [inline backend/CircleForms/Models/MongoDbRange.cs:3]: Change this and other similar classes into `record`s?

**Review by stanriders — CHANGES_REQUESTED** [review]

**Comment by stanriders** [comment]
You should update Dockerfile as well

File: backend/CircleForms/Models/MongoDbRange.cs (hunk 2 above)
Comment (JustRoxy) [reply backend/CircleForms/Models/MongoDbRange.cs:3]: records due to `{ init; }` can break serialization and it's not required for range to be natively comparable

**Review by stanriders — APPROVED** [review]

**Pull request merged**